.prettierignore

```diff
@@ -0,0 +1,18 @@
+# Ignore assets
+src/assets
+src/stories/assets
+
+# Ignore fixtures
+src/stories/fixtures
+cem/test/fixtures
+
+# Ignore ESLint config file
+**/.eslintrc.cjs
+
+# Ignore MD files
+**/*.md
+
+# Ignore stories
+#src/components/*/*.stories.js
```

Comment:
I'm in favor of that: We should not ignore stories

Reply:
Let's add them (even if it makes them a bit more verbose)